src/TestSuite/TestCase.php

```diff
@@ -39,16 +39,16 @@
 use Exception;
 use LogicException;
 use Mockery;
+use Mockery\Adapter\Phpunit\MockeryTestCase;
```

Comment:
or it should use the `MockeryPHPUnitIntegration` and `MockeryTestCaseSetUp` traits